### PR TITLE
Deprecate fedora.client.bindings.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: fedora/client/bodhi.py [hunk 1]
```diff
@@ -17,6 +17,10 @@
 # License along with python-fedora; if not, see <http://www.gnu.org/licenses/>
 #
 """
+DEPRECATED: This module is deprecated and has moved to bodhi.client.bindings.
+As the bodhi-0.9 client still uses it, it will remain here until Fedora 25 is
+EOL.
+
```
Comment (pypingou) [inline fedora/client/bodhi.py:23]: Should we just output a deprecation warning?

Comment (bowlofeggs) [reply fedora/client/bodhi.py:23]: @pypingou I considered that, but I don't think I'd like the bodhi client in F24/F25 to be emitting deprecation warnings if I can help it. I think what I'd like to do is to stick with this message for now, then once F27 is released we can add in a deprecation warning, and then after n months we can delete this module. What do you think?

Comment (pypingou) [reply fedora/client/bodhi.py:23]: Well, depends if we push the update to the stable branch of Fedora :)

Comment (bowlofeggs) [reply fedora/client/bodhi.py:23]: @pypingou I don't intend to do that - it is backwards incompatible and I don't want to assume that developers haven't scripted against the client. I plan to just let Fedora 25 go EOL with bodhi 0.9, and then consider 0.9 fully retired.

Comment (pypingou) [reply fedora/client/bodhi.py:23]: I meant the python-fedora update, if we only push an update with the deprecation warning in rawhide, we should be fine :)

**Comment by bowlofeggs** [comment]
On Thu, 2017-02-23 at 09:52 -0800, Pierre-Yves Chibon wrote:
> I meant the python-fedora update, if we only push an update with the
> deprecation warning in rawhide, we should be fine :)

Oh sure, we could do it that way if you prefer. I wasn't sure what the
release procedures were for this package. In this case, I will rework
this PR to raise a deprecation warning.

**Comment by bowlofeggs** [comment]
@pypingou I added a deprecation warning, what do you think?

**Comment by pypingou** [comment]
Looks fine to me :)

**Pull request merged**